packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -434,7 +434,7 @@ export default class extends EntityModel {
     );
 
     await this.createOutgoingLink(graphApi, {
-      linkTypeModel: WORKSPACE_TYPES.linkType.ofOrg,
+      linkTypeModel: WORKSPACE_TYPES.linkType.hasMembership,
```

Comment:
The validation introduced in this PR helped me catch this mistake that slipped in a while ago 🙃